src/tests/utils/convert.js

```diff
@@ -1,20 +1,31 @@
 require('should');
+const requireFromString = require('require-from-string');
 const convert = require('../../utils/convert');
 const definitions = {
   basic: require('./definitions/basic.json'),
+  basicScripting: require('./definitions/basic-scripting.json'),
   apiHeader: require('./definitions/api-header.json'),
   apiQuery: require('./definitions/api-query.json'),
   session: require('./definitions/session.json'),
   oauth2: require('./definitions/oauth2.json'),
+  oauth2Scripting: require('./definitions/oauth2-scripting.json'),
```

Comment:
`oauth2-scripting.json` is same as `oauth2` except that it defines `pre_oauthv2_token` and `post_oauthv2_token` method in scripting.